docs/new-guide-how-to-provide-your-own-platform.md

```diff
@@ -0,0 +1,88 @@
+---
+id: new-guide-how-to-provide-your-own-platform
+title: How to Provide Your Own Platform
+---
+
+:::note What is this guide about?
+This guide will show you how to offer your own platform to application teams using meshStack. This enables you to provide platforms in an easy fashion including cost management, lifecycle management, self-service provisioning and enforce a compliant level of security and governance.
+:::
+
+
+On a high level, you need the following and your platform integration is done:
+
+1. Create a platform with its own name and icon.
+2. Create a Building Block that does the automation in the background using Terraform
+3. Create a Landing Zone that does the roll out of the Building Block.
+
+## Prerequisites
+
+- You have access to the Platform Builder in meshStack.
+- You have the necessary platform credentials and configuration details.
+- Terraform knowledge is helpful, as meshStack relies on Terraform for custom platform integrations.
+- There is some kind of platform/tenancy organization model in the cloud platform, just like e.g. Azure has an AAD (platform) and Subscriptions (tenant).
+- The above can be created & automated with Terraform.
```

Comment:
```suggestion
- The above can be created & automated with e.g. Terraform.
```

Reply:
I will add a note that its also possible with other measures in the beginning of the guide